endpoints/petrol/index.js

```diff
@@ -0,0 +1,38 @@
+import request from 'request'
+import h from 'apis-helpers'
+import app from '../../server'
+
+function queryData(callback) {
+  const url = 'https://raw.githubusercontent.com/gasvaktin/gasvaktin/master/vaktin/gas.min.json'
```

Comment:
See where you're coming from @koddsson and it's a valid point – but this is actually not very different from what we were thinking of doing with APIs v2, except the method of fetching the data from the external package.. Downside is we lose the relative safety of source control (breaking changes will effectively break the APIs endpoint, but scraping a website has the same problem really)..

Not sure which way I'm leaning, while I like the idea of having the relative safety of "owning" the scraper within APIs, I'd be lying if I didn't admit that the idea of arbitrary sources being used, using arbitrary programming languages / techniques etc, sounds nice. I feel it reflects on the endless possibilities of this project..

Hey @Loknar, first of all thanks for the contribution! I actually agree with you that using this kind of source would be a more stable solution compared to scraping gsmbensin.is. The git-history of price changes sounds like a nice idea (has the price really not changed in 5 days though?), only thing discouraging me there is the possibility of accidentally automatically pushing a broken version of the json file, but that is something that could easily be handled with some tests..

I think what @koddsson was referring to regarding backdoors is exactly what you pointed out, malicious intent – it's more of an in theory point, and perhaps more in the context of this setting a precedent for allowing this without thought in the future.. But honestly that is always a possibility when we're scraping data from arbitrary websites IMHO..

I'd love to hear what the other kristjáns think about this @kristjanmik @MiniGod